sdk/react/bottomOverlay.js

```diff
@@ -100,6 +100,12 @@ var BottomOverlay = React.createClass({
     }
   },
 
+  componentWillReceiveProps: function(nextProps) {
+    if (Math.abs(this.props.playhead - nextProps.playhead) >= 0.2) {
+       this.setState({playhead:-1.0}); 
```

Comment:
can we rename playhead?  it makes the this.state.playhead and this.props.playhead confusing.

maybe something like cachedPlayhead, seeking Playhead, etc.